modules/remove_book.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
+export const removeBook = (booklist, index, save, load) => {
```

Comment:
- [ ] I suggest that you should not disable any `eslint` guidelines set for this project because `eslint` check clean code and javascript best practices usage :blush: I think `removeBook` is the only function you are exporting in this file so eslint is recommending for you to use the `export default` :relaxed: